.github/workflows/tests.yml

```diff
@@ -98,6 +98,18 @@ jobs:
         build-root-directory: run/ktor-hello-world
         wrapper-directory: run/ktor-hello-world
 
+    - name: Cloud Run (Micronaut) - Gradle
+      uses: eskatos/gradle-command-action@v1
+      with:
+        arguments: test --console=plain --info
+        build-root-directory: run/micronaut-hello-world
+        wrapper-directory: run/micronaut-hello-world
+
+    - name: Cloud Run (Quarkus) - Gradle
+      run: |
+        cd ${GITHUB_WORKSPACE}/run/quarkus-hello-world
+        ./mvnw test
```

Comment:
Another nit: let's use `./mvnw test -q`, otherwise we get 5K+ lines of output instead of ~100